projects/core/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mat-datetimepicker/core",
-  "version": "15.0.0",
+  "version": "15.0.1",
```

you don't have to increase the versions. the release job will do that for us.

```suggestion
  "version": "15.0.0",
```

---

Oh very clevers! Didn't know that.